ui/src/pages/Lore/lore.scss

```diff
@@ -4,6 +4,9 @@
   width: 80%;
   height: 70vh;
   margin: 0 auto;
+  &__no_effect {
```

Comment:
This should be &__noEffect

Reply:
Nah that messes with the snake case.

Reply:
@jmorales8

Reply:
get rid of this. I dont see us using this in the future

Reply:
ok